.docker/gui-docker

```diff
@@ -6,13 +6,15 @@
 
 # All arguments to this script except "-c <container_name>" will be appended to a docker run command.
 # If a container name is specified, and this container already exists, the container is re-entered,
-# which easily allows to run persistent containers.
+# which easily allows entering the same persistent container from multiple terminals.
 
 # Example commands:
 # ./gui-docker --rm -it moveit/moveit:master-source /bin/bash     # Run a (randomly named) container that is removed on exit
-# ./gui-docker -v ~/ros_ws:/root/ros_ws --rm -it moveit/moveit:master-source /bin/bash   # Same, but also link host volume ~/ros_ws to /root/ros_ws
-# ./gui-docker -c different_container_name                        # Start (or continue) an interactive bash in a moveit/moveit:master-source container
+# ./gui-docker -v ~/ros_ws:/root/ros_ws --rm -it moveit/moveit:master-source /bin/bash   # Same, but also link host volume ~/ros_ws to /root/ros_ws in the container
+# ./gui-docker -c container_name                                  # Start (or continue) an interactive bash in a moveit/moveit:master-source container
 # ./gui-docker                                                    # Same, but use the default container name "default_moveit_container"
+# ./gui-docker -c container_name -v ~/ros_ws:/root/ros_ws -it moveit/moveit:master-source /bin/bash  # Same, but also link host volume ~/ros_ws to /root/ros_ws in the container
```

Comment:
I explicitly dropped this example, because you cannot run this twice in a row:
The second call doesn't allow any options, but just will re-enter the already running container.

I can image though, to add an example like this:
`gui-docker -c container_name <command>`
which runs the command (instead of bash) in non-interactive fashion within an existing container.

Reply:
Because it's too complicated to explain the interplay of the initial and subsequent gui-docker commands, I suggest dropping the example here. However, in the documentation (https://github.com/ros-planning/moveit.ros.org/pull/282) we can provide the required explanations and the example.

Reply:
Good with me, I pushed the change.